Assets/Scripts/Buildings/BuildingBlueprint.cs

```diff
@@ -21,7 +21,11 @@ public class BuildingBlueprint : MonoBehaviour
         [Tooltip("How strong the blueprint should be faded out")] [SerializeField] [Range(0f, 1f)]
         private float fadeStrength;
 
-        public Building Building => building;
+        public Building Building
```

Comment:
Can be simplified to:
```suggestion
        public Building Building { get; set; }
```

Reply:
kk

Reply:
nope, this suggestion does not work and causes exceptions

Reply:
Okay, nvm, my mistake 😅 Didn't recognise the backing field